platform/src/main/java/org/hillview/table/api/IColumn.java

```diff
@@ -116,4 +116,9 @@ default String getName() {
     }
 
     default ContentsKind getKind() { return this.getDescription().kind;}
+
+    /**
+     * @return A hash code for the item in the rowIndex. Exception if it is missing.
+     */
+    long hashCode64(int rowIndex, long seed);
```

Comment:
I think that the API should not have a seed; the seed is part of the XX construction.
Unlike Randomness, it is probably OK to use a singleton static XX as the "global" XX function.
So you have a static method to set the seed at the very beginning. You also have another static method to get the XX, which replaces LongHashFunction.xx(seed).